platforms/android/include.gradle

```diff
@@ -38,3 +38,5 @@ dependencies {
 //    compile "com.google.android.gms:play-services-auth:9.6.+"
 
 }
+
+apply plugin: "com.google.gms.google-services"
```

Comment:
Could you double check if it works when you add by hand this in platform/android:
```
  dependencies {
    classpath "com.android.tools.build:gradle:X.X.X"
    classpath "com.google.gms:google-services:3.0.0"
  }
```

Without the:
```
apply plugin: "com.google.gms.google-services"
```